test/JDBC/input/storedProcedures/Test-sp_addrolemember-vu-prepare.sql

```diff
@@ -0,0 +1,12 @@
+-- tsql
+CREATE ROLE sp_addrolemember_r1;
+GO
+
+CREATE ROLE sp_addrolemember_r2;
+GO
+
+CREATE LOGIN sp_addrolemember_l1 WITH PASSWORD = '123';
+GO
+
+CREATE USER sp_addrolemember_u1 FOR LOGIN sp_addrolemember_l1;
```

Comment:
I'm not sure if `CREATE USER` will work as expected in older version (13.4) because we didn't support BBF user at that time. Please avoid creating user in prepare script if you still want to run the upgrade test in 13.4->latest MVU.